src/background.js

```diff
@@ -6,15 +6,21 @@
 
 'use strict';
 
+import { makeLogger } from './lib/utils';
+const log = makeLogger('BE');
+
 import moment from 'moment';
+import 'moment/min/locales.min';
+browser.i18n.getAcceptLanguages().then(languages => {
```

Comment:
Could this `getAcceptLanguages()` dance get done once in `init()` up in `src/popup/snooze-popup.js`, rather than in every component? Then, also pass in a ref to `getMessage()` alongside other functions like `scheduleSnoozedTab()`?